src/applications/personalization/profile360/components/AddressEditModal.jsx

```diff
@@ -1,11 +1,17 @@
 import React from 'react';
 
+import {
+  FIELD_NAMES
+} from '../constants/vet360';
+
 import {
   consolidateAddress,
   expandAddress
```

Comment:
We need to get rid of `expandAddress` here and in the `onSubmit`. This one is from the old Letters implementation; but we do our own address inference here: https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/applications/personalization/profile360/actions/misc.js#L99

The expandAddress in this form helper also uses the wrong type for military address; Vet360 expects `MILITARY OVERSEAS` instead of `MILITARY`

Reply:
Yeah that makes sense, I didn't know if there was still some need for that transformation. We transitioned from PCIU to Vet360 addresses in a blur, and I wasn't sure how well I followed the new logic. 

I feel like this is out of the scope of this PR though. Do you think it makes more sense to tackle this during the upcoming refactor?

Reply:
Agreed; I will add it to that issue